src/App.tsx

```diff
@@ -53,18 +56,21 @@ const App: React.FC = () => (
       </>
     )}
 
-    <UserProvider>
-      <CookiesProvider>
-        <ThemeProvider theme={mdTheme}>
-          <CssBaseline />
-          <Router>
-            <Routes>
-              <Route path="/*" element={<ControlPanel />} />
-            </Routes>
-          </Router>
-        </ThemeProvider>
-      </CookiesProvider>
-    </UserProvider>
+    <QueryClientProvider client={queryClient}>
+      <UserProvider>
+        <CookiesProvider>
+          <ThemeProvider theme={mdTheme}>
+            <CssBaseline />
+            <Router>
+              <Routes>
+                <Route path="/*" element={<ControlPanel />} />
+              </Routes>
+            </Router>
+          </ThemeProvider>
+        </CookiesProvider>
+      </UserProvider>
+      <ReactQueryDevtools initialIsOpen={false} />
```

Comment:
add its devtools. this shows up as a small icon in the bottom right of the screen during dev and is not shown on built versions. 
![{CF962667-F812-42E1-904C-A399BE8D291C}](https://github.com/user-attachments/assets/fa392c97-52a5-4202-acdd-0c8a27423e96)
it gives you a view into the cache and buttons to test loading/error states
![image](https://github.com/user-attachments/assets/a7605063-4f9d-4e7e-8afb-af5940121a9f)